### DISCUSSION TITLE
[Feature] Offload backedup photos

### DISCUSSION THREAD

**Discussion by ldehner** [immich-app/immich#12300] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A feature on the apps just like iclouid does on its own. Opt in feature that backed up photos which were not accessed in the last x days will be automatically deleted of the device and only a preview will remain on the device. When the photo is viewed it will be downloaded again from the server

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#165
